packages/design-tokens/tokens/shadow/box-shadow.json

```diff
@@ -0,0 +1,13 @@
+{
+  "box-shadow": {
+    "s": {
+      "value": "0px 2px 10px 0px rgba(0, 0, 0, 0.07)"
```

Comment:
We could have a bit more realistic shadow by using multiple shadows. 
But it comes with a minor performance hit.
https://www.joshwcomeau.com/css/designing-shadows/

Reply:
Interesting idea. What do you think @PetriLahdelma ?